packages/ui-components/src/components/Map/__tests__/markerFormats.test.js

```diff
@@ -185,6 +185,25 @@ describe('measures', () => {
       const display3 = getMeasureDisplayInfo({ questionC: 0.5 }, [optionsSpectrum]);
       expect(display3).toHaveProperty('color', 'hsl(50, 100%, 50%)');
     });
+
+    it('Coloured radius choose a colour from a spectrum, and pick radius from radius', () => {
```

Comment:
This test title is a little hard to understand. I suggest we call it something shorter like 'should support coloured radius' and if you want the longer description pop in in a comment inside the test case